doc/manual/src/command-ref/nix-store.md

```diff
@@ -156,6 +156,14 @@ To test whether a previously-built derivation is deterministic:
 $ nix-build '<nixpkgs>' -A hello --check -K
 ```
 
+To see the stderr/stdout of a build:
+
+```console
+$ nix-store --read-log $(nix-instantiate ./test.nix)
+```
+
+Use `nix log` to see the logs of a build.
+
```

Comment:
````suggestion
Alternatively, use [`nix log`](./new-cli/nix3-nix-log.md) (experimental) to show the build logs:

```console
nix log $(nix-instantiate ./test.nix)
```
````

To be honest, I find it a bit hard to integrate it smoothly here, so I'd also be fine with removing the line completely: 

```suggestion
```